index.html

```diff
@@ -80,41 +84,110 @@ <h2 class="display-4">Who we are</h2>
   <section>
     <div class="container">
       <div class="row align-items-center">
-        <div class="col-md-4">
+        <div class="col-md-4 order-2">
           <div class="p-5 d-none d-md-block">
-            <img class="img-fluid rounded-circle" src="./images/history.jpg" alt="">
+            <img class="img-fluid rounded-circle " src="./images/NY.jpg" alt="">
           </div>
         </div>
-        <div class="col-md-8">
+        <div class="col-md-8 order-1">
           <div class="p-5">
-            <h2 class="display-4">Proud of our history</h2>
```

Comment:
Why did we need to remove this paragraph? I think a link to the blog post describing our history is worth keeping.

Reply:
It was a mistake, typo in the template. The "Proud of our history" section in back